core/templates/dev/head/pages/exploration_editor/editor_tab/StateSolution.js

```diff
@@ -52,6 +52,7 @@ oppia.controller('StateSolution', [
         stateInteractionIdService.savedMemento,
         explorationStatesService.getInteractionCustomizationArgsMemento(
           EditorStateService.getActiveStateName()),
+          true,
```

Comment:
Deindent by 2, this is not connected to the previous arg.

Reply:
Done